### PR TITLE
REL: 0.9.0

### PR DESCRIPTION
**Pull request by adelavega**
I think it's about time for a release.

I've included #276 in this so let's wait to merge that, first.

Also want to wait on #278 if it looks easy.

I updated the CHANGELOG and .zenodo.

I couldn't find @leej3's ORCID. Added @jdkent (please confirm affiliation ;)

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -49,6 +49,25 @@
...
+    {
+      "name": "Gorgolewski, Krzysztof J.",
+      "affiliation": "Department of Psychology, Stanford University",
+      "orcid": "0000-0003-3321-7583"
+    },
+    {
+      "name": "Gorgolewski, Krzysztof J.",
+      "affiliation": "Department of Psychology, Stanford University",
+      "orcid": "0000-0003-3321-7583"
+    },
```
Comment (effigies) [inline .zenodo.json:62]: Too many Krzysztofs.

```suggestion
```

File: .zenodo.json [hunk 2]
```diff
@@ -49,6 +49,25 @@
...
+    {
+      "affiliation": "University of Texas at Austin",
+      "name": "Kent, James D.",
+      "orcid": "0000-0002-4892-2659"
+    },
+    {
+      "affiliation": "National Institute of Mental Health",
+      "name": "Lee, John"
```
Comment (effigies) [inline .zenodo.json:70]: Can you put these above Russ? Russ + Chris are "final authors".

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,20 @@
...
+This release added AFNI's 3dREMLfit for first-level estimation, dropped nistats
+in favor of nilearn.stats, and several other minor improvements.
+
+* ENH: disable automatic drift modeling (https://github.com/poldracklab/fitlins/pull/276)
+* ENH: modify path patterns to make task optional (https://github.com/poldracklab/fitlins/pull/275)
+* MNT: Update PyBIDS and nilearn (https://github.com/poldracklab/fitlins/pull/268)
+* FIX: AFNI contrast bool (https://github.com/poldracklab/fitlins/pull/266)
+* CI: Make faster? (https://github.com/poldracklab/fitlins/pull/264)
+* TST: Add dummy dataset for pytest/local testing (https://github.com/poldracklab/fitlins/pull/262) 
+* ENH: Add AFNI 3dREMLfit for first-level estimation (https://github.com/poldracklab/fitlins/pull/171)
```
Comment (effigies) [inline CHANGES.md:15]: Can you reorder FIX ENH TST MNT CI (otherwise keeping order)?

**Comment by effigies** [comment]
Thanks. Long overdue. AFNI's big enough I'd make it a minor release (0.9.0) over a bug-fix.

File: .zenodo.json (hunk 1 above)
Comment (adelavega) [reply .zenodo.json:62]: But he's so cool

**Comment by adelavega** [comment]
I also thought 0.9.0 would be fair given how big this release is. Let's go w/ that.

**Comment by adelavega** [comment]
Okay, made the changes. Let's wait on all my new PRs from today though.

**Review by jdkent — APPROVED** [review]
LGTM (my affiliation is correct :)

**Comment by Shotgunosine** [comment]
Can we hold the release for just a day or two? I’ve almost got testing
against reference results ready to submit.

**Comment by yarikoptic** [comment]
I just want to say THANK YOU @adelavega @effigies @Shotgunosine et al.
I have not participated recently, and have not looked inside or tried it yet (will do for sure!), **BUT** it is great to see fitlins going forward.

**Comment by adelavega** [comment]
@effigies I updated the CHANGES file to reflect @Shotgunosine 's merged PR.

Once tests pass this should be good to go right? Merge, and push tagged commit, and edit release on GH should do it right?

**Comment by effigies** [comment]
Yes, merge, tag with -a, push, and update GH release.

**Comment by Shotgunosine** [comment]
I don’t have any more changes for this release, thanks for reviewing that
monster PR.

**Comment by adelavega** [comment]
Hmm, tests seem to have regressed, will try to re-run to see if they're just flakey

**Comment by Shotgunosine** [comment]
I think this is because the reference datasets were generated before the drift option was added. I'll update the tests to specify cosine drift and see if that fixes things.

**Comment by Shotgunosine** [comment]
@adelavega, tests are passing now

**Comment by adelavega** [comment]
Awesome, time to release!

**Pull request merged**